libpromises/generic_agent.c

```diff
@@ -446,6 +446,20 @@ void LoadAugments(EvalContext *ctx, GenericAgentConfig *config)
     free(def_json);
 }
 
+void AddPolicyEntryVariables (EvalContext *ctx, GenericAgentConfig *config)
```

Comment:
constify parameters that are not modified

Reply:
Done. I also made the function ``static``.